### PR TITLE
Update python packages.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@peterbe r?

File: sugardough/requirements.txt [hunk 1]
```diff
@@ -3,15 +3,11 @@
 # sha256: D3VN0_wgThq7_XnEfyVZXFqU0kfOWuUrsmmZWqLnmTM
 peep==2.1
 
-# TarGZ
```
Comment (peterbe) [inline sugardough/requirements.txt]: Thank you! Keeping those labels up to date was going to be annoying

**Comment by peterbe** [comment]
r+

**Comment by glogiotatidis** [comment]
merci!

**Pull request merged**